docs/index.rst

```diff
@@ -54,6 +54,7 @@ method, this part of the documentation is for you.
   modules/activation
   modules/models
   modules/distributed
+  modules/db
```

Comment:
@luomai should we release db now?

Reply:
I didn't found how to deactivate Sphinx from trying to build it. I had an error, maybe there is a way ;)

Reply:
We are currently refactoring the DB module.To avoid this error, a possible way is to keep the DB doc and highlight this module is under construction.